package/MDAnalysis/core/topologyattrs.py

```diff
@@ -1416,8 +1416,7 @@ class Angles(_Connection):
     """Angles between three atoms
 
     Initialise with a list of 3 long tuples
-    Eg:
-      [(0, 1, 2), (1, 2, 3), (2, 3, 4)]
+    Eg: [(0, 1, 2), (1, 2, 3), (2, 3, 4)]
```

Comment:
"E.g.," contains periods and is followed by a comma. think it makes more sense to show the list of tuples as code.